server/controllers/articles.js

```diff
@@ -30,15 +30,21 @@ class ArticleController {
       const articleDetails = await validateArticle(req.body);
       const { id: userId } = req.decoded;
       const { title } = articleDetails;
+      let { tags } = articleDetails;
       articleDetails.title = articleDetails.title.replace(/ +/g, ' ');
       const createArticleDetails = {
         slug: title, userId, ...articleDetails
       };
-
+      if (typeof tags === 'undefined' || tags === null) {
```

Comment:
is this needed?

`const tags = articleDetails.tags || [];`

Reply:
you are absolutely right. I understand now. Thanks.